app/src/main/java/com/kickstarter/KSApplication.java

```diff
@@ -60,6 +66,16 @@ public void onCreate() {
       .build();
     component().inject(this);
 
+    /*
+      setting cookie for unique id tracking, hardcoding prod
+     */
+    final HttpCookie cookie = new HttpCookie("vis", UUID.randomUUID().toString());
+    cookie.setMaxAge(DateTime.now().plusYears(100).getMillis());
```

Comment:
geologically speaking, is 100 years into the future really that distant? :trollface:

Reply:
should we go into _the next millenium_